src/api/python/cvc5.pxi

```diff
@@ -735,23 +740,11 @@ cdef class Solver:
         term.cterm = self.csolver.mkSepNil(sort.csort)
         return term
 
-    def mkString(self, str_or_vec):
+    def mkString(self, str s):
         cdef Term term = Term(self)
-        cdef vector[unsigned] v
-        if isinstance(str_or_vec, str):
-            for u in str_or_vec:
-                v.push_back(<unsigned> ord(u))
-            term.cterm = self.csolver.mkString(<const vector[unsigned]&> v)
-        elif isinstance(str_or_vec, list):
-            for u in str_or_vec:
-                if not isinstance(u, int):
-                    raise ValueError("List should contain ints but got: {}"
-                                     .format(str_or_vec))
-                v.push_back(<unsigned> u)
-            term.cterm = self.csolver.mkString(<const vector[unsigned]&> v)
-        else:
-            raise ValueError("Expected string or vector of ASCII codes"
-                             " but got: {}".format(str_or_vec))
+        cdef Py_ssize_t size
+        cdef const wchar_t* tmp = PyUnicode_AsWideCharString(s, &size)
```

Comment:
This is a little out of my scope of understanding, but do you think you need to free this with `PyMem_Free`?  I'm looking at the documentation here: https://docs.python.org/3/c-api/unicode.html

Reply:
As this is the first time I'm using this stuff: I have no idea :-)
I'll have a look...

Reply:
Yeah, it seems we should do this.